ReportSimulationOutput/measure.rb

```diff
@@ -1443,7 +1504,15 @@ def write_timeseries_output_results(runner, outputs, output_format,
 
     if output_format == 'csv'
       # Assemble data
-      data = data.zip(*fuel_data, *end_use_data, *emissions_data, *hot_water_use_data, *total_loads_data, *comp_loads_data, *zone_temps_data, *airflows_data, *weather_data)
+      if timestamps_dst && timestamps_utc
+        data = data.zip(timestamps2, timestamps3, *fuel_data, *emissions_data, *end_use_data, *hot_water_use_data, *total_loads_data, *comp_loads_data, *zone_temps_data, *airflows_data, *weather_data)
+      elsif timestamps_dst
+        data = data.zip(timestamps2, *fuel_data, *end_use_data, *emissions_data, *hot_water_use_data, *total_loads_data, *comp_loads_data, *zone_temps_data, *airflows_data, *weather_data)
+      elsif timestamps_utc
+        data = data.zip(timestamps3, *fuel_data, *end_use_data, *emissions_data, *hot_water_use_data, *total_loads_data, *comp_loads_data, *zone_temps_data, *airflows_data, *weather_data)
+      else
+        data = data.zip(*fuel_data, *end_use_data, *emissions_data, *hot_water_use_data, *total_loads_data, *comp_loads_data, *zone_temps_data, *airflows_data, *weather_data)
+      end
```

Comment:
This is pretty silly, you shouldn't have to do this. I mean, we already handle cases where some outputs are empty and some are populated because of what the user has requested. You should be able to figure out how to handle timestamps in a similar way without all these conditionals.